frontend/packages/dev-console/src/components/topology/TopologyPage.tsx

```diff
@@ -32,10 +38,17 @@ const getTopologyActiveView = () => {
   return localStorage.getItem(LAST_TOPOLOGY_VIEW_LOCAL_STORAGE_KEY);
 };
 
+const setTopologyWorkloadsActiveView = (id: string) => {
+  localStorage.setItem(LAST_TOPOLOGY_WORKLOADS_VIEW_LOCAL_STORAGE_KEY, id);
+};
+
+const getTopologyWorkloadsActiveView = () => {
+  return localStorage.getItem(LAST_TOPOLOGY_WORKLOADS_VIEW_LOCAL_STORAGE_KEY);
+};
```

Comment:
Does UX want a separate setting for the topology page and for the workload page?

Reply:
UX and PM agreed to using a single setting.

Reply:
Sorry, missed the thought behind this. We are keeping view preference separate per sections (Workloads vs Topology). UX and PM agreed to that. The saved layout of the graph view is shared between the sections.